ros_buildfarm/release_job.py

```diff
@@ -93,6 +93,13 @@ def configure_release_jobs(
         set(pkg_names) & set(build_file.package_ignore_list)
     explicitly_ignored_pkg_names = \
         set(pkg_names) - set(filtered_pkg_names) - explicitly_ignored_without_recursion_pkg_names
+
+    upstream_disabled_pkg_names = set(
+        p for r in dist_file.repositories.values()
+        if r.release_repository and not r.release_repository.version
+        for p in r.release_repository.package_names)
+    explicitly_ignored_pkg_names |= upstream_disabled_pkg_names
```

Comment:
```suggestion
    # Get package names for which the release version is missing, indicating the release has been "disabled" in the distribution file.
    upstream_disabled_pkg_names = set(
        p for r in dist_file.repositories.values()
        if r.release_repository and not r.release_repository.version
        for p in r.release_repository.package_names)
    explicitly_ignored_pkg_names |= upstream_disabled_pkg_names
```

This may just be an artifact of reviewing this as the very first thing I'm doing this morning but since there's a compound generator expression involved I think a comment establishing the intent of this "query" is warranted.

But also it's a really nice generator expression :man_cook: :ok_hand: